src/dispatch/static/dispatch/src/app/store.js

```diff
@@ -65,7 +65,7 @@ const mutations = {
     state.loading = value
   },
   RESET_SNACKBAR(state) {
-    state.snackbar = Object.assign(state.snackbar, getDefaultSnackbarState())
+    state.snackbar = Object.assign({}, getDefaultSnackbarState())
   },
   RESET_REFRESH(state) {
     state.refresh = Object.assign(state.refresh, getDefaulRefreshState())
```

Comment:
We may want to do the same here, replace `state.refresh` with `{}`, but this doesn't seem to be causing any issues that I can tell.